rs/BUILD.bazel

```diff
@@ -80,3 +80,13 @@ package_group(
         "//rs/state_tool/...",
     ],
 )
+
+# Boundary node code.
+# Can depend on: ic-protocol package groups
+# Dependants: //rs:release-pkg.
+package_group(
+    name = "boundary-node-pkg",
```

Comment:
@rumenov how will this package group be used?

Reply:
it restricts visibility. So it should make it harder for example the replica to depend on the BN and vice-versa.